.github/workflows/cppcheck.yml

```diff
@@ -6,11 +6,9 @@ on:
 jobs:
   build_and_test:
     runs-on: ubuntu-latest
+    container:  borbelytomii/tests-image:latest
     steps:
       - uses: actions/checkout@v2
-      - run: sudo apt-get install cppcheck
-      - run: sudo apt-get install valgrind
-      - run: cppcheck *.cpp --enable=all --output-file=cppcheck_for_artifact.txt
```

Comment:
gondolom ezt betettetek command-nak. szerintem szebb lenne ha itt maradna, mert a docker image dockergfile-jat kell megnezni, hogy mi a rakot csinal.

Reply:
Megneztem, nem tettetek be, szoval szeintem ez a command ugy ahgy van, kitorlodott.

Reply:
Igaz, javítva.